guides/inventory/stock-items.md

```diff
@@ -0,0 +1,48 @@
+# Stock items
+
+On-hand inventory is tracked using the `Spree::StockItem` model. Each
+`Spree::Variant` in a store has a corresponding `Spree::StockItem` object with a
+`count_on_hand` value that represents the number of items you have in stock.
```

Comment:
> Each `Spree::Variant` in a store has a corresponding `Spree::StockItem` object

it has a corresponding `Spree::StockItem` object for each stock location present in the store. So the same variant can have more than one `Spree::StockItem` associated objects.